packages/app/src/app/components/Markdown/Link.tsx

```diff
@@ -28,13 +28,9 @@ export const LinkElement = ({ href, children, ...props }) => {
     );
   }
 
-  if (!href.includes('codesandbox')) {
-    return (
-      <Link target="_blank" rel="noopener noreferrer ugc" {...props}>
-        {children}
-      </Link>
-    );
-  }
-
-  return <Link {...props}>{children}</Link>;
+  return (
```

Comment:
This wasn't to allow links that are outside links but just to add the rel="noopener noreferrer ugc"  to only links for outside codesanbdox